src/components/PeopleList/PeopleList.tsx

```diff
@@ -0,0 +1,43 @@
+import cn from 'classnames';
+import React from 'react';
+import { Person } from '../../types/Person';
+import './PeopleList.scss';
+
+type Props = {
+  people: Person[];
+  onSelect: (name: string) => void;
+};
+
+export const PeopleList: React.FC<Props> = (({
+  people,
+  onSelect = () => {},
+}) => {
+  return (
+    <div className="dropdown-menu" role="menu">
+      <div className="dropdown-content">
+        {people.length ? people.map((person: Person) => (
+          <button
```

Comment:
it'd be nice to reset the default button styles or use another tag:
![image](https://github.com/mate-academy/react_autocomplete/assets/86923089/b16b2870-4801-4fc1-84ad-1b9b640c55f8)